flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -54,31 +58,31 @@ object AggregateUtil {
     * organized by the following format:
     *
     * {{{
-    *                   avg(x) aggOffsetInRow = 2          count(z) aggOffsetInRow = 5
-    *                             |                          |
-    *                             v                          v
-    *        +---------+---------+--------+--------+--------+--------+
-    *        |groupKey1|groupKey2|  sum1  | count1 |  sum2  | count2 |
-    *        +---------+---------+--------+--------+--------+--------+
+    *                          avg(x)                             count(z)
+    *                             |                                   |
+    *                             v                                   v
+    *        +---------+---------+-----------------+------------------+------------------+
+    *        |groupKey1|groupKey2|  AvgAccumulator |  SumAccumulator  | CountAccumulator |
+    *        +---------+---------+-----------------+------------------+------------------+
     *                                              ^
     *                                              |
-    *                               sum(y) aggOffsetInRow = 4
+    *                                           sum(y)
     * }}}
     *
     */
   private[flink] def createPrepareMapFunction(
       namedAggregates: Seq[CalcitePair[AggregateCall, String]],
       groupings: Array[Int],
       inputType: RelDataType)
-    : MapFunction[Row, Row] = {
```

Comment:
Ok, based on the DEV discussion, I thought we are going to fix the code style in the new PRs. But you are right, I should just keep the style changes in the new lines. I will discard the "style only" change.